modular_skyrat/modules/synths/code/bodyparts/power_cord.dm

```diff
@@ -93,6 +93,10 @@
 		user.nutrition += power_use / SYNTH_CHARGE_PER_NUTRITION
 		do_sparks(1, FALSE, target_apc)
 
+	if(target_apc.main_status == APC_HAS_POWER)
```

Comment:
Actually tested this myself because I was curious, and this PR is something I've wanted to happen. When synths charge from APC's it never updates the status to `APC_LOW_POWER` It always remains at a value of 2 which is `APC_HAS_POWER`. Even when it starts recharging with this PR the status remains at `APC_HAS_POWER` Even when it should be well past the low power point.  But even still, it coulden't hurt to set it to `if(target_apc.main_status <= APC_HAS_POWER)` Which should cover all bases

<details>
<summary>Screenshot</summary>

![dreamseeker_nhYrVpn4Op](https://github.com/Skyrat-SS13/Skyrat-tg/assets/2568378/7d3cea6f-7101-47b8-9aef-894a82fd06a8)


</details>

Reply:
Added `if(target_apc.main_status <= APC_HAS_POWER)` alongside update appearance suggestion, seems to be working fine.

<details>
<summary>additional testing video</summary>

https://github.com/Skyrat-SS13/Skyrat-tg/assets/60776130/5f739a61-9cdd-44a7-be39-fcea4d3605a0

</details>